modules/users/client/utils/networks.js

```diff
@@ -2,34 +2,30 @@
  * Determine if given user handle for Warmshowers is an id or username
  * @link https://github.com/Trustroots/trustroots/issues/308
  */
-function isWarmshowersId(profile) {
+export function isWarmshowersId(id) {
   // WarmShowers id should contain only digits
-  return /^[0-9]*$/.test(profile.extSitesWS);
+  return /^[0-9]*$/.test(id);
 }
 
 /**
  * Check if there are additional accounts
  */
-function hasConnectedAdditionalSocialAccounts(profile) {
-  return (profile.additionalProvidersData && Object.keys(profile.additionalProvidersData).length);
+export function hasConnectedAdditionalSocialAccounts(profile) {
+  return profile.additionalProvidersData && Object.keys(profile.additionalProvidersData).length > 0;
 }
 
 /**
  * Return an URL for user's social media profiles
  * Ensure these values are published at users.profile.server.controller.js
  */
-function socialAccountLink(providerName, providerData) {
+export function socialAccountLink(providerName, providerData) {
   if (providerName === 'facebook' && providerData.id) {
-    return 'https://www.facebook.com/app_scoped_user_id/' + providerData.id;
+    return `https://www.facebook.com/app_scoped_user_id/${providerData.id}`;
   } else if (providerName === 'twitter' && providerData.screen_name) {
-    return 'https://twitter.com/' + providerData.screen_name;
+    return `https://twitter.com/${providerData.screen_name}`;
   } else if (providerName === 'github' && providerData.login) {
-    return 'https://github.com/' + providerData.login;
+    return `https://github.com/${providerData.login}`;
   } else {
     return '#';
   }
 }
-
-
-
-module.exports = { hasConnectedAdditionalSocialAccounts, socialAccountLink, isWarmshowersId };
```

Comment:
We should probably be careful with this kind of object exports. It works because in ES6 I can pull keys from an object. I don't think these are equal in terms of code-splitting some day:

```js
const foo;
export { foo };
// ...
import { foo } from '.';
```

```js
export const foo;
// ...
import { foo } from '.';
```

...not entirely sure about this tho and didn't start dig deeper.